operator/tests/common/central-cr-with-scanner-v4-assert.yaml

```diff
@@ -30,7 +30,7 @@ collectors:
 - command: retry-kubectl.sh describe pod -n $NAMESPACE -l app=scanner-v4-matcher
 - command: retry-kubectl.sh describe pod -n $NAMESPACE -l app=scanner-v4-db
 # Please keep the above lists in sync with pods-debug.yaml
-timeout: 1500
+timeout: 3600 # Need to account for vuln definition load
```

Comment:
I think the agreement in the CI improvements meeting was to stop checking matcher health (and do it later in the other test jobs)?

Reply:
Yes that's right. I haven't gotten to implement the changes we discussed in the call yet, unfortunately.